run.sh

```diff
@@ -0,0 +1,13 @@
+git clone https://github.com/appwrite/appwrite
```

Comment:
We don't need this file. A simple `composer install` on the readme should be enough.

Reply:
okay